src/v2/view-builder/views/ov/EnrollChannelPollDescriptionView.js

```diff
@@ -41,33 +41,45 @@ export default View.extend({
                 {{i18n code="oie.enroll.okta_verify.setup.customUri.noPrompt"
                 bundle="login" $1="<span class='semi-strong'>$1</span>"}}
               </div>
-              <div class="desktop-instructions">
-                {{i18n code="oie.enroll.okta_verify.setup.customUri.makeSureHaveOV" bundle="login"}}
-              </div>            
+              {{#unless sameDeviceOVEnrollmentEnabled}}
+                <div class="desktop-instructions">
+                  {{i18n code="oie.enroll.okta_verify.setup.customUri.makeSureHaveOV" bundle="login"}}
+                </div>
+              {{/unless}}
             {{else}}
               {{i18n code="oie.enroll.okta_verify.setup.customUri.makeSureHaveOVToContinue" bundle="login"}}
             {{/if}}
           </p>      
           <ol class="ov-info">
-            {{#if deviceMap.platformLC}}
+            {{#unless sameDeviceOVEnrollmentEnabled}}
+              {{#if deviceMap.platformLC}}
+                <li>
+                  <a aria-label='{{i18n code="customUri.required.content.download.linkText" bundle="login"}}'
+                  href="{{deviceMap.downloadHref}}"
+                  class="app-store-logo {{deviceMap.platformLC}}-app-store-logo"></a>
+                </li>
+              {{/if}}            
+                          
               <li>
-                <a aria-label='{{i18n code="customUri.required.content.download.linkText" bundle="login"}}'
-                href="{{deviceMap.downloadHref}}"
-                class="app-store-logo {{deviceMap.platformLC}}-app-store-logo"></a>
-              </li>
-            {{/if}}            
-                        
-            <li>
-              {{i18n code="oie.enroll.okta_verify.setup.customUri.setup"
-              bundle="login" $1="<span class='semi-strong'>$1</span>"}}
-            </li>            
-            
+                {{i18n code="oie.enroll.okta_verify.setup.customUri.setup"
+                bundle="login" $1="<span class='semi-strong'>$1</span>"}}
+              </li>     
+            {{/unless}}
             <li>
               <a href="{{deviceMap.setupOVUrl}}" class="button button-primary setup-button">
               {{i18n code="oie.enroll.okta_verify.setup.title" bundle="login"}}
               </a>
             </li>
 
+            {{#if sameDeviceOVEnrollmentEnabled}}
+              <li>
+                <span>{{i18n code="customUri.required.content.download.title" bundle="login"}}</span>&nbsp;
+                <a href="{{deviceMap.downloadHref}}" target="_blank" id="download-ov" class="orOnMobileLink">
```

Comment:
Not a big deal, but does the `download-ov` ID serve any additional purpose besides being used as a selector in the page object? If not, could we just use the `orOnMobileLink` class as the selector?

Reply:
not sure, just following existing pattern of how the link is constructed